src/login.ts

```diff
@@ -89,17 +89,11 @@ export function loginFromSavedState(saved_state: any) {
     });
 
     // This try/catch might be able to detect invalid saved state, as
-    // firebase will probably throw an error
+    // it will probably throw an error
     try {
-      const user = new notion.api.firebase.app.firebase_.User(
```

Comment:
@pwstegman What do you think?

Reply:
Here's the issue discussing the workaround for Firebase login persistence in Node: https://github.com/firebase/firebase-js-sdk/issues/1874#issuecomment-549085119

They use the `User` method, so we'll want to find where it's officially exposed and use it